gsy_framework/sim_results/all_results.py

```diff
@@ -21,7 +21,9 @@
 
 class ResultsHandler:
     """Calculate all results for each market slot."""
+
     def __init__(self, should_export_plots: bool = False, is_scm: bool = False):
+        self.forward_market_enabled = False
```

Comment:
I do not see this member being used. Can you please help me understand why it is needed?

Reply:
This is something which probably could be improve. I added this attribute to differentiate between `ForwardResultsHandler` and `ResultsHandler`, to know if we are using forward market or not. It helps to tell in gsy-web which ResultsHandler should be used like [here](https://github.com/gridsingularity/gsy-web/blob/41fcacfdd8a8cffff2d3bc3fb071b8ba5d7ff206/simulation_results/results_calculation_helper.py#L94) or [here](https://github.com/gridsingularity/gsy-web/blob/41fcacfdd8a8cffff2d3bc3fb071b8ba5d7ff206/simulation_results/results_calculation_helper.py#L52).

Reply:
you could use the following on the gsy-web side.
```python
isinsttance( self.simulation_results[data["job_id"]], ForwardResultsHandler)
```
But the fact that you have to add an if-block there shows to me, that the interface of the `ResultsHandler` and `ForwardResultsHandler` are not the same. This can probably be improved. Let me further review to check this out.

Reply:
So should I change the approach with the additional attribute `self.forward_market_enabled`  for both classes and check `isinsttance` whenever needed?